src/sbvr-api/sbvr-utils.ts

```diff
@@ -1569,7 +1570,7 @@ const runPut = (
 	_res: _express.Response,
 	request: uriParser.ODataRequest,
 	tx: _db.Tx,
-) => {
+): Promise<undefined> => {
```

Comment:
Why not `void`?

Reply:
Because the callers expect `undefined | number | db.Result` - this is just to explicitly type the result with the value it's currently implicitly typed with

Reply:
Cool, I thought it would be due to what the callers expect, but I expected the result was communicated simply via the promise being resolved or rejected, in which case `void` was more appropriate.

Reply:
Yeah, in this case success/failure is what really matters but there's other `runX` methods that return real values so the `undefined` is to distinguish that